wownar-react-sdk/src/config.ts

```diff
@@ -0,0 +1,8 @@
+// use NODE_ENV to not have to change config based on where it's deployed
+export const NEXT_PUBLIC_URL =
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:3000'
+    : 'https://onchain-app-template.vercel.app';
+// Add your API KEY from the Coinbase Developer Portal
+export const NEXT_PUBLIC_CDP_API_KEY = process.env.NEXT_PUBLIC_CDP_API_KEY;
```

Comment:
Would you be able to add a corresponding comment in the README that the developer would have to get credentials from Coinbase Developer Platform when filling out their .env file?